snitun/client/client_peer.py

```diff
@@ -119,14 +121,32 @@ async def start(
         )
 
         # Task a process for pings/cleanups
-        self._loop.create_task(self._handler())
+        assert not self._handler_task, "SniTun connection already running"
```

Comment:
Is this assertion for developers during development?

Reply:
Yes